bunsen/bunsen-avro/src/test/java/com/cerner/bunsen/avro/AvroConverterTest.java

```diff
@@ -266,10 +266,9 @@ public void testSingleReference() {
 
     Record subject = (Record) avroCondition.get("subject");
 
-    Assert.assertEquals(testCondition.getSubject().getReference(),
-        subject.get("reference"));
+    Assert.assertEquals(testCondition.getSubject().getReference(), subject.get("reference"));
 
-    Assert.assertEquals("12345",  subject.get("PatientId"));
```

Comment:
Made the changes in test file. Although, I am checking on lower case of this string

Reply:
Sorry, can you clarify this please? I just want to make sure I understand the root cause. My question was that I was expecting that _not_ updating the R4 version of this tests, should fail them. IOW, `subject.get("PatientId")` in the corresponding R4 test will return `null` and hence the `assertEquals` fails. I understand that you have updated it now, but why didn't it fail before?
[making a direct comment here since the file is renamed]

Reply:
The previous code in DefinitionToAvroVisitor > RelativeValueConverter > fromHapi was:
`url.startsWith(prefix)`

But then I changed it to:
`uri.toLowerCase().startsWith(prefix.toLowerCase()`
and this is the reason it was not failing earlier

Even now it should not fail even though I have made the changes as such in the test file cause the current change is something similar:
`lowercase(uri).startsWith(lowercase(prefix))`

Reply:
I see, thanks.